packages/shared/lib/ledger.ts

```diff
@@ -1,67 +1,62 @@
 import { get, writable } from 'svelte/store'
```

Comment:
I think we should refactor this file into a new `lib/core/ledger` module :raised_hands: